portal-impl/test/unit/com/liferay/portal/test/rule/FeatureFlagTestRuleTest.java

```diff
@@ -24,6 +28,16 @@ public class FeatureFlagTestRuleTest {
 	public static final AggregateTestRule aggregateTestRule =
 		LiferayUnitTestRule.INSTANCE;
 
+	@BeforeClass
+	public static void setUpClass() throws PortalException {
+		String featureFlag = "feature.flag.METHOD-456";
```

Comment:
This variable is used once, no need to declare it, you can hard code it.

Reply:
@thiago-buarqque The reason why I did it this way is because SF catches it otherwise:
```
[java] 3: Use annotation "@FeatureFlags" instead of "PropsUtil.addProperties" for feature flag, see https://github.com/liferay/liferay-portal/blob/master/modules/util/source-formatter/src/main/resources/documentation/check/java_test_method_annotations_check.markdown: ./portal-impl/test/unit/com/liferay/portal/test/rule/FeatureFlagTestRuleTest.java 33 (SourceCheck:JavaTestMethodAnnotationsCheck)
     [java]
     [java] 	at com.liferay.source.formatter.SourceFormatter.format(SourceFormatter.java:472)
     [java] 	at com.liferay.source.formatter.SourceFormatter.main(SourceFormatter.java:301)
[16:15](https://liferay.slack.com/archives/D06JRQHBPLP/p1731683751540309)
```

SF checks if there is a string starting with "feature.flag" after PropsUtil.addProperties(

Reply:
@brbalazs ah I see. In that case I think you can do `FeatureFlagConstants.FEATURE_FLAG + METHOD-456` instead of having the variable. 

Or does the SF also catch that case? :thinking:

Reply:
@thiago-buarqque it's working what you suggested, thanks!